packages/commands/src/utils.mjs

```diff
@@ -37,22 +60,84 @@ export const initializeAudiusSdk = async ({
     })
   )
 
-  if (!audiusSdk) {
+  if (!audiusSdk || !currentUserId || (handle && currentHandle !== handle)) {
+    // If handle was provided, unset current entropy and replace with the entropy
+    // for the given user before initializing UserAuth
+    if (handle) {
+      localStorage.removeItem('hedgehog-entropy-key')
```

Comment:
So having to use localStorage directly now because that was nested in libs before right?

Reply:
Sort of. There's some special logic in audius-cmd because we use localStorage to manage entropy for multiple users and let you change the active one using the `--from` flag. Otherwise, we wouldn't have to do anything at all with localStorage and could just use Hedgehog login/logout functionality.